odra-macros/Cargo.toml

```diff
@@ -11,11 +11,11 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
-odra-utils = { path = "../utils", features = ["std"] }
 proc-macro2 = "1.0.69"
 quote = "1.0.33"
 syn = { version = "2.0.29", features = ["full"] }
 syn_derive = "0.1.8"
+convert_case = { version = "0.5.0" }
```

Comment:
Not needed if using odra-core

Reply:
This crate is not no-std, it's simpler this way

Reply:
Ok.